presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -101,7 +101,6 @@ public void testCodepoint()
     @Test
     public void testConcat()
     {
-        assertInvalidFunction("CONCAT()", "There must be two or more concatenation arguments");
```

Comment:
Maybe leave the empty arguments test so that we know the code doesn't fail in some fancy way?